src/Faker/Provider/uk_UA/Address.php

```diff
@@ -268,7 +278,6 @@ class Address extends \Faker\Provider\Address
         'Урицького',
         'П. Орлика',
         'Хрещатик',
-        'Артема',
```

Comment:
I can't find the list of streets in the URL given in phpDoc. So I can't check that the list is up to date, or that you've removed that name for a good reason. Please explain why you removed that line or revert.